### ISSUE TITLE
2012 core installation stalls after windows update

### ISSUE THREAD

**Issue by byrdog55** [mwrock/packer-templates#16]
It seems like boxstarter is not surviving the first reboot that occurs after windows update on the 2012 core install.  Am I the only only one having this issue?

**Comment by mwrock** [comment]
That's right. Core was a poc that did not get far. Boxstarter restarts aftrt reboot by creating a file in the startup programs folder. Core does not look in this folder for programs to start. So I need to change boxstarter to accommodate that.

**Comment by howardjones** [comment]
I just ran into this (I think) - having gotten my gui server just right, I had assumed I would be able to just set the 'core' variable :-/

Has anything changed since 2015? What is the necessary change, if I need to do that myself?

**Comment by howardjones** [comment]
For anyone following, logging in by hand and running 

    cmd.exe /c C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe -File a:\boxstarter.ps1

again seems to kick everything off again to get to the stage where winrm is enabled. For my use-case (rare rebuilds) this should be enough.
